src/Server.ts

```diff
@@ -0,0 +1,62 @@
+import * as express from "express"
+import * as methods from "./router"
+const bodyParser = require("body-parser")
+
+import * as routes from "./router"
+import {Controller} from "./controller/trainee/Controller"
+import {errorHandler} from "./libs/routes/errorHandler"
+import router from "./controller/trainee/routes"
+// import { resolveMx } from "dns"
+
+let controllerDao = new Controller()
+
+export class ServerDao
+{
+        public path ="/api"
+        private app; PORT ; NODE_ENV
+    constructor(config){
+        this.app=express()
+        this.PORT = config.PORT,
+        this.NODE_ENV = config.NODE_ENV
+
+    }
+
+    bootstrap(){
+        this.initBodyParser()
+       return this.setUpRoutes()
+    }
+
+    setUpRoutes(){
+        try{
+            // controllerDao.get('/api/health-check', function(req,res,next){
+                
+            // })
+            this.app.use(this.path, routes);
+            // this.app.use(errorHandler)
+
+            console.log("sucesss")
+        }
+        catch(error){
+            throw error;
+            
+        }
+    }
+
+    run(){
+        try{
+            this.app = express()
```

Comment:
don't you think it should be in the constructor?

Reply:
i did not get you as i am calling run method from **index.ts**